pymc/gp/gp.py

```diff
@@ -233,15 +234,16 @@ def conditional(self, name, Xnew, given=None, jitter=JITTER_DEFAULT, **kwargs):
 
         Parameters
         ----------
-        name: string
+        name : str
             Name of the random variable
-        Xnew: array-like
-            Function input values.
-        given: dict
-            Can optionally take as key value pairs: `X`, `y`,
-            and `gp`.  See the section in the documentation on additive GP
+        Xnew : array-like
+            Function input values. If one-dimensional, must be a column
+            vector with shape `(n, 1)`.
+        given : dict, optional
+            Can take as key value pairs: `X`, `y`,
+            and `gp`. See the section in the documentation on additive GP
```

Comment:
This can't be a mention, it should be a cross-reference. Anyone knows what page it means?

Reply:
After digging around for a while, it looks like the only illustration of how additive GPs work is https://www.pymc.io/projects/examples/en/latest/gaussian_processes/GP-MaunaLoa.html

It seems like a gap in the examples/docs and might need to be addressed in future work.

Reply:
Maybe it's this page still pending update from v3? https://github.com/pymc-devs/pymc/issues/5720 cc @bwengals @canyon289

Reply:
Can we link to unfinished work in a way that the docs will sync up once #5720 is finished or should we leave this be for now?

Reply:
Yeah it might be, it should point here: https://www.pymc.io/projects/docs/en/v3/Gaussian_Processes.html#additive-gps